bin/additional_tests/test_launch_interpret_json.py

```diff
@@ -0,0 +1,128 @@
+        d = {
+            "experiment": "MyCustomExperiment",
+            "custom": [
+                {
+                    "noise": [
+                        {
+                            "column_name": "input1",
+                            "name": "UniformTextMasker",
+                            "params": {"probability": 0.1}
+                        },
+                        {
+                            "column_name": "input2",
+                            "name": "GaussianNoise",
+                            "params": {"mean": 0.5, "std": 0.1}
+                        }],
+                    "split": [
+                        {
+                            "name": "RandomSplitter",
+                            "params": {"split": [0.6, 0.4, 0]}
+                        }]},
+                {
+                    "noise": [
+                        {
+                            "column_name": "input2",
+                            "name": "UniformTextMasker",
+                            "params": {"probability": 0.1}
+                        },
+                        {
+                            "column_name": "float",
+                            "name": "GaussianNoise",
+                            "params": {"mean": 0.5, "std": 0.1}
+                        }],
+                    "split": [
+                        {
+                            "name": "RandomSplitter",
+                            "params": {"split": [0.6, 0.8, 0.1]}
+            }]}]}
+        
+        out_l = [{'experiment': 'MyCustomExperiment', 'noise': None, 'split': None},
+        {'experiment': 'MyCustomExperiment', 'noise': [{'column_name': 'input1', 'name': 'UniformTextMasker', 'params': {'probability': 0.1}}, {'column_name': 'input2', 'name': 'GaussianNoise', 'params': {'mean': 0.5, 'std': 0.1}}], 'split': [{'name': 'RandomSplitter', 'params': {'split': [0.6, 0.4, 0]}}]},
+        {'experiment': 'MyCustomExperiment', 'noise': [{'column_name': 'input2', 'name': 'UniformTextMasker', 'params': {'probability': 0.1}}, {'column_name': 'float', 'name': 'GaussianNoise', 'params': {'mean': 0.5, 'std': 0.1}}], 'split': [{'name': 'RandomSplitter', 'params': {'split': [0.6, 0.8, 0.1]}}]}]
+        self.assertEqual(interpret_json(d), out_l)
+
+    
+    def test_interpret_json_with_column_wise(self):
+        d = {
+            "experiment": "MyCustomExperiment",
+            "interpret_parmas_mode": "culumn_wise", 
+            "noise": [
+                {
+                    "column_name": "hello:input1:dna",
+                    "name": ["UniformTextMasker", "AnotherNoiser", "AnotherNoiser"],
+                    "params": [{"probability": [0.1, 0.2]}, {"probability": [0.11, 0.21]}, {"mean": [0.5, 0.6], "std": [0.1, 0.2]}]
+                },
+                {
+                    "column_name": "hello:input2:prot",
+                    "name": ["UniformTextMasker", "YetAnotherNoiser"],
+                    "params": "default"
```

Comment:
I though here you will have to say `"params":["default", "default"]` for the column_wise option to match "UniformTextMasker" and "YetAnotherNoiser"